genesis/version4/interfaces/bmad.py

```diff
@@ -0,0 +1,317 @@
+from ..input._lattice import Quadrupole, Corrector, Drift, Marker, Undulator
+from ..input._main import Setup, Field, Track, Beam
+
+
+from pmd_beamphysics.units import mec2
+
+from scipy.constants import c
+
+from math import pi, sqrt
+import numpy as np
+
+
+def label_from_bmad_name(bmad_name: str) -> str:
+    """
+    Formats a label by standardizing case, removing backslashes, and replacing disallowed characters.
+
+    Parameters
+    ----------
+    bmad_name : str
+        The Bmad name string to be formatted.
+
+    Returns
+    -------
+    str
+        A formatted label suitable for Genesis4.
+    """
+    name = bmad_name.upper()
+    label = name.split("\\")[-1]  # Extracts text after backslash, if any
+    return label.replace(".", "_").replace("#", "_")
```

Comment:
This is for superimposed elements like 'P1/Q2'.

Multipass elements end with numbers like: `cav\1`, but those are not implemented here.

The tao.ele_lord_slave return is confusing to me, this will have to be a TODO item to generalize.